TechTalk.JiraRestClient/JiraClient.cs

```diff
@@ -163,11 +168,21 @@ public Issue<TIssueFields> CreateIssue(String projectKey, String issueType, TIss
                 if (issueFields.timetracking != null)
                     issueData.Add("timetracking", new { originalEstimate = issueFields.timetracking.originalEstimate });
 
-                var propertyList = typeof(TIssueFields).GetProperties().Where(p => p.Name.StartsWith("customfield_"));
-                foreach (var property in propertyList)
+                var propertyInfos = typeof(TIssueFields).GetProperties().ToArray();
+                var propertiesFromAttribute = propertyInfos.Select(p => new { Property = p, FieldAttribute = p.GetAttribute<FieldAttribute>() })
+                    .Where(a => a.FieldAttribute != null)
+                    .Select(p => new NamedProperty(p.Property, p.FieldAttribute.FieldName));
+
+                var customFields = propertyInfos.Where(p => p.Name.StartsWith("customfield_")).Select(p => new NamedProperty(p, p.Name));

```

Comment:
line is rather long, makes sense to introduce line breaks before '.Where' and '.Select'